itou/templates/apply/includes/profile_infos.html

```diff
@@ -143,10 +143,34 @@ <h3 class="mb-0">Situation</h3>
             {% endif %}
         </li>
         <li>
-            <small class="align-self-start">Autres</small>
-            {% if other_infos %}
-                <strong>
+            <small>Autres</small>
+            {% if not other_infos %}
```

Comment:
Ce `{% if not other_infos %} ... {% elif other_infos|length > 1 %} ... {% else %} ... {% endif %}` me semble un peu lourd avec la répétiion des `{% if item == "" %}` . Si y'a moyen de faire plus light , je suis preneur